org.keepassxc.KeePassXC.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "org.keepassxc.KeePassXC",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "keepassxc",
+  "rename-icon": "keepassxc",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc", "--socket=x11",
+    /* Wayland access */
+    "--socket=wayland",
+    /* Access user files */
+    "--filesystem=home",
+    /* Notification access */
+    "--talk-name=org.freedesktop.Notifications",
+    /* Screen Lock Listener */
+    "--talk-name=org.freedesktop.login1.Manager",
+    "--talk-name=org.freedesktop.ScreenSaver",
+    "--talk-name=org.gnome.SessionManager",
+    "--talk-name=org.gnome.SessionManager.Presence",
+    "--talk-name=com.canonical.Unity.Session",
+    /* KeePassHTTP */
+    "--share=network",
+    /* YubiKey USB access */
+    "--device=all"
+  ],
+  "cleanup": [
+    "*.a",
+    "*.h",
+    "*.la",
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "keepassxc",
+      "buildsystem": "cmake",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DWITH_TESTS=OFF",
+        "-DWITH_XC_HTTP=ON",
+        "-DWITH_XC_YUBIKEY=ON"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/keepassxreboot/keepassxc/releases/download/2.2.2/keepassxc-2.2.2-src.tar.xz",
+          "sha256": "2f55784f7e6d3a840575e1768a3e5ae03986c2f6a70029dd6b9f9bb0fa443473"
+        },
+        {
+          /* Backport of upstream patch */
+          "type": "patch",
+          "path": "keepassxc-id-fix.patch"
+        }
+      ],
+      "post-install": [
+        "for f in COPYING LICENSE*; do install -Dm644 $f /app/share/licenses/org.keepassxc.KeePassXC/$f; done"
+      ],
+      "cleanup": [ "keepassxc-cli" ],
+      "modules": [
+        {
+          "name": "libusb",
+          "config-opts": [ "--disable-static", "--disable-udev", "--prefix=/app" ],
+          "sources": [{
+            "type": "git",
+            "url": "https://github.com/libusb/libusb.git",
+            "branch": "v1.0.21"
+          }],
+          "post-install": [
+            "install -Dm644 COPYING /app/share/licenses/libusb/COPYING"
+          ]
+        },
+        {
+          "name": "libyubikey",
+          "sources": [
+            {
+              "type": "git",
+              "url": "https://github.com/Yubico/yubico-c.git",
+              "branch": "libyubikey-1.13"
+            },
+            {
+              "type": "shell",
+              "commands": [ "autoreconf --install" ]
```

Comment:
I prefer this but its a minor detail:
```json
{
  "type": "script",
  "dest-filename": "autogen.sh",
  "commands": ["autoreconf --install"]
}
```

Reply:
I usually prefer that too. Not sure why I did not do it this time.